src/filesystem/drive/directory_handle.rs

```diff
@@ -693,8 +693,19 @@ impl DirectoryHandle {
 
         let mut inner_write = self.inner.write().await;
         let node = inner_write.by_perm_id_mut(&new_permanent_id).await?;
-        let node_data = node.data_mut().await;
+        let mime_type = {
+            let node_name = node.name().clone();
+            MimeGuesser::default()
+                .with_name(node_name)
+                // .with_data(content_references.map())
```

Comment:
Is the plan to revisit using the "magic byte detection" and rely on extensions for now?